test/boolean_test.cpp

```diff
@@ -189,11 +187,9 @@ TEST(Boolean, CoplanarProp) {
   MeshGL cylinderGL = WithIndexColors(cylinder.GetMesh());
   cylinder = Manifold(cylinderGL);
 
-  Manifold cylinder2 = cylinder.Scale({0.5f, 0.5f, 1.0f})
-                           .Rotate(0, 0, 15)
-                           .Translate({0.25f, 0.25f, 0.0f});
+  Manifold cylinder2 = cylinder.Scale({0.8f, 0.8f, 1.0f}).Rotate(0, 0, 5);
   Manifold out = cylinder - cylinder2;
-  ExpectMeshes(out, {{42, 84, 3, 68}});
+  ExpectMeshes(out, {{52, 104, 3, 88}});
```

Comment:
I finally figured out why CoplanarProp was giving inconsistent results between my Mac and our CI: the test case is dependent on how many of the top-face triangles get intersected in the Boolean op. Our new triangulator is less deterministic (more dependent on vert sorting order). I fixed the test by making it so no matter what circle triangulation is used, every edge will always get intersected, which makes the test result deterministic, even when the triangulation is not.